### PR TITLE
News card&scrollbar

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
Good for me :100:

File: src/Components/NewsCard/style.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  flex-direction: column;
+  margin: 25px;
+  width: 300px;
+  height: 400px;
+  img {
+    height: 150px;
+  }
+  h1 {
+    margin: 20px 0 0 0;
+
```
Comment (Hacene010) [inline src/Components/NewsCard/style.jsx]: empty line to remove here !

File: src/Components/NewsCard/style.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    font-size: 14px;
+    color: grey;
+  }
+  p.newsContent {
+    font-size: 20px;
+    margin: 10px 0 0 0;
+  }
+
```
Comment (Hacene010) [inline src/Components/NewsCard/style.jsx:26]: same here

File: src/Components/NewsCard/style.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  p.newsSource {
+    position: relative;
+    bottom: 0;
+  }
+  a {
+    text-decoration: none;
+    color: var(--color--secondary);
+
```
Comment (Hacene010) [inline src/Components/NewsCard/style.jsx]: here too

File: src/Components/NewsCard/style.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    margin: 0;
+
```
Comment (Hacene010) [inline src/Components/NewsCard/style.jsx]: : /

File: src/Components/NewsCard/style.jsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    :hover {
+      color: var(--color--secondary);
+    }
+  }
+
```
Comment (Hacene010) [inline src/Components/NewsCard/style.jsx:40]: -_-

**Review by Hacene010 — CHANGES_REQUESTED** [review]
some evils

**Review by Caroline-CD — APPROVED** [review]
OK :100: it's ok for me

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**